Traducir.Web/src/App/Components/Filters.tsx

```diff
@@ -201,7 +201,7 @@ export default class Filters extends React.Component<IFiltersProps, IFiltersStat
         </>;
     }
 
-    public hasFilter() {
+    public hasFilter(): any | undefined {
```

Comment:
Unfortunatly that is a bug since it will never be true, none of the variables of status is boolean so === (triple) will be always false.

Reply:
indeed :) then, something like this could work

```
const result = this.state.sourceRegex ||
             this.state.translationRegex ||
             this.state.key ||
             this.state.translationStatus ||
             this.state.suggestionsStatus ||
             this.state.pushStatus ||
             this.state.urgencyStatus;
return !!result;
```